### PR TITLE
[doc] Use correct parameters for Presto & Trino

### PR DESCRIPTION
**Pull request by lvyanquan**
### Purpose
file.format is defined here
https://github.com/apache/incubator-paimon/blob/41997b2170dcfa32aa867bbc4573989911f05f61/paimon-common/src/main/java/org/apache/paimon/CoreOptions.java#L91
bucket-key
https://github.com/apache/incubator-paimon/blob/41997b2170dcfa32aa867bbc4573989911f05f61/paimon-common/src/main/java/org/apache/paimon/CoreOptions.java#L67
changelog-producer
https://github.com/apache/incubator-paimon/blob/41997b2170dcfa32aa867bbc4573989911f05f61/paimon-common/src/main/java/org/apache/paimon/CoreOptions.java#L373
primary_key & partitioned_by shouldn't be changed 
https://github.com/apache/incubator-paimon-trino/blob/75360d8e0b11cf5026f5219d0c9af02afee2b984/paimon-trino-common/src/main/java/org/apache/paimon/trino/TrinoTableOptions.java#L36

Linked issue: close #xxx

### CODE REVIEW

File: docs/content/engines/presto.md [hunk 1]
```diff
@@ -124,12 +124,12 @@ CREATE TABLE paimon.test_db.orders (
     order_date date
 )
 WITH (
-    file_format = 'ORC',
+    file.format = 'ORC',
```
Comment (JingsongLi) [inline docs/content/engines/presto.md:127]: Have you tested this?

Comment (leaves12138) [reply docs/content/engines/presto.md:127]: You can run paimon-build:
mvn spotless:apply && mvn clean install -DskipTests && mvn package -Pgenerate-docs -pl paimon-docs -nsu -DskipTests && cd paimon-core && mvn test && cd ../paimon-flink && mvn test && cd ../paimon-docs && mvn test && cd ..
in paimon base directory to test this

Comment (JingsongLi) [reply docs/content/engines/presto.md:127]: I mean, Trino has made special treatment because "." or "-" is not allowed in trino.

Comment (lvyanquan) [reply docs/content/engines/presto.md]: Thanks for pointing this and suggestion, after testing and find out that those parameters don't need change.

截屏2023-08-03 下午3 23 04

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**